### ISSUE TITLE
Race condition when getting the id of the new REPL window

### ISSUE THREAD

**Issue by jnturton** [mreppen/kakoune-sway#5]
On my machine, this runs a little bit before Sway has updated the focus and I had to hack in a `sleep 0.1s` in front of it.
```
winid=$(swaymsg -t get_tree | jq '.. | select(.type?) | select(.focused==true) | .id')
```
I looked for a while for better ways than using focused==true to get winid but I couldn't come up with anything good using things like `$PPID` in the shell script to try get the terminal emulator pid for use in a different jq query.

**Comment by mreppen** [comment]
Hi @dzamo, I suspect your terminal emulator starts executing before it creates a window.

I implemented another approach in 1ee7704b98c457f67d050443b9ea31bd3f09078c on the `get_id` branch. Would you give it a go and see how it works?

That approach listens to sway for new windows and saves the id for whatever is the first created window.

**Comment by jnturton** [comment]
It works :+1:
